src/modules/form/components/DynamicDisplay.tsx

```diff
@@ -51,17 +46,18 @@ const DynamicDisplay: React.FC<Props> = ({
           severity={SeverityMap[item.component as keyof typeof SeverityMap]}
           sx={{ mt: 2 }}
         >
-          <div dangerouslySetInnerHTML={html} />
+          <CommonHtmlContent>{html}</CommonHtmlContent>
```

Comment:
Changes in this file aren't related to rich-text labels, but just using the `CommonHtmlContent` component for consistency